### ISSUE TITLE
Update the ND image to 9.0.5.9

### ISSUE THREAD

**Issue by majguo** [WASdev/azure.websphere-traditional.cluster#110]
I found that tWAS 9.0.5.9 is available and we should update the twas-nd and ihs images to that level.

Here is the log info which demonstrated `9.0.5.9` is available now:
```
[root@ihsa6188fVM ~]# tail -f /var/log/cloud-init-was.log
Checking at + Sun Sep 26 10:14:32 UTC 2021
                 25%                50%                75%                100%
------------------|------------------|------------------|------------------|
............................................................................
Updated to com.ibm.java.jdk.v8_8.0.6036.20210913_1556 in the /datadrive/IBM/WebSphere/IHS/V9/ directory.
Updated to com.ibm.websphere.IHS.v90_9.0.5009.20210823_0038 in the /datadrive/IBM/WebSphere/IHS/V9/ directory.
Updated to com.ibm.java.jdk.v8_8.0.6036.20210913_1556 in the /datadrive/IBM/WebSphere/Plugins/V9/ directory.
Updated to com.ibm.websphere.PLG.v90_9.0.5009.20210823_0038 in the /datadrive/IBM/WebSphere/Plugins/V9/ directory.
Updated to com.ibm.java.jdk.v8_8.0.6036.20210913_1556 in the /datadrive/IBM/WebSphere/Toolbox/V9/ directory.
Updated to com.ibm.websphere.WCT.v90_9.0.5009.20210823_0038 in the /datadrive/IBM/WebSphere/Toolbox/V9/ directory.
Entitled

[root@manageda6188fVM1 ~]# tail -f /var/log/cloud-init-was.log
Checking at + Sun Sep 26 10:14:21 UTC 2021
                 25%                50%                75%                100%
------------------|------------------|------------------|------------------|
............................................................................
Updated to com.ibm.java.jdk.v8_8.0.6036.20210913_1556 in the /datadrive/IBM/WebSphere/ND/V9/ directory.
Updated to com.ibm.websphere.ND.v90_9.0.5009.20210823_0038 in the /datadrive/IBM/WebSphere/ND/V9/ directory.
Installed 9.0.5.7-WS-WAS-IFPH38929_9.0.5007.20210907_0921 to the /datadrive/IBM/WebSphere/ND/V9/ directory.
Entitled

[root@dmgra6188fVM ~]# tail -f /var/log/cloud-init-was.log
Checking at + Sun Sep 26 10:14:16 UTC 2021
                 25%                50%                75%                100%
------------------|------------------|------------------|------------------|
............................................................................
Updated to com.ibm.java.jdk.v8_8.0.6036.20210913_1556 in the /datadrive/IBM/WebSphere/ND/V9/ directory.
Updated to com.ibm.websphere.ND.v90_9.0.5009.20210823_0038 in the /datadrive/IBM/WebSphere/ND/V9/ directory.
Installed 9.0.5.7-WS-WAS-IFPH38929_9.0.5007.20210907_0921 to the /datadrive/IBM/WebSphere/ND/V9/ directory.
Entitled
```

**Comment by majguo** [comment]
Refer to [howto-update-for-was-fixpack](https://github.com/WASdev/azure.websphere-traditional.cluster/blob/master/docs/howto-update-for-was-fixpack.md) for the image and solution update.

**Comment by git4rk** [comment]
- twas-nd image: https://github.com/WASdev/azure.websphere-traditional.image/actions/runs/1282665858

**Comment by git4rk** [comment]
@majguo @edburns Question regarding this step:
![image](https://user-images.githubusercontent.com/24283162/135109125-bbabe3ea-f462-45a2-b657-cef2131b5380.png)

Do I need to update these? Where do I get those values from?
```
       <twasnd.image.version>9.0.20210726</twasnd.image.version>
        <ihs.image.version>9.0.20210726</ihs.image.version>
```

**Comment by git4rk** [comment]
@gcharters Can you please execute the Partner Center steps 4.iii and 4.iv of https://github.com/WASdev/azure.websphere-traditional.cluster/blob/master/docs/howto-update-for-was-fixpack.md#updating-the-image
See URL details here: https://github.com/WASdev/azure.websphere-traditional.cluster/issues/110#issuecomment-929280797

**Comment by majguo** [comment]
> * Do I need to update these in Solution's pom.xml? Where do I get those values from?
> 
> ```
>        <twasnd.image.version>9.0.20210726</twasnd.image.version>
>         <ihs.image.version>9.0.20210726</ihs.image.version>

Yes. image versions in Solution's `pom.xml` must be updated as long as images are updated in the partner center. They're `Disk version` you specified in the previous step (steps 4.iii and 4.iv, search "Specify a new value for Disk version and write it down" to find where it is)

> * What does this version number indicate (Solution's pom.xml)? Is this visible to customers?
>   `<version>1.3.20</version>`

Partner center requires version of package file for the Solution bumped every time the solution is updated, this version number is designed for this purpose. See step 3.ii of https://github.com/WASdev/azure.websphere-traditional.cluster/blob/master/docs/howto-update-for-was-fixpack.md#updating-and-publishing-the-solution-code. Generally it's invisible to customer as the version number now is added as parameter of Issue tracker in the home page of offer, and you can only see it when hovering over the link but it will be removed from address bar if you clicking the link due to the ms aka link configuration. See this in the on-going [PR #111](https://github.com/WASdev/azure.websphere-traditional.cluster/pull/111/commits/52ddbf2eb7b4f7548894caa6b193a40dd50792b3#diff-13ef47a38ffa55984e721db0089f0973ce81a67dfea9f34f20032baa2ef82e64R69).

**Comment by git4rk** [comment]
Graham completed the image publishing steps. Version number is `9.0.20210930`.

**Comment by git4rk** [comment]
Updated version # : https://github.com/WASdev/azure.websphere-traditional.cluster/pull/114

**Comment by git4rk** [comment]
step 3: completed by Graham. He also made updates to the instructions: https://github.com/WASdev/azure.websphere-traditional.cluster/pull/115
step 4: Created release https://github.com/WASdev/azure.websphere-traditional.cluster/releases/tag/1.3.25

**Comment by git4rk** [comment]
Preview: https://azuremarketplace.microsoft.com/en-us/marketplace/apps/ibm-usa-ny-armonk-hq-6275750-ibmcloud-aiops.2021-04-08-twas-cluster-preview?tab=Overview&flightCodes=159d4370-8565-48a3-ba5d-7ed76f64cd8f

Failed with:

{"code":"InvalidTemplateDeployment","details":[{"code":"InvalidParameter","target":"imageReference","message":"The following list of images referenced from the deployment template are not found: Publisher: ibm-usa-ny-armonk-hq-6275750-ibmcloud-aiops, Offer: 2021-06-03-ihs-base-image, Sku: 2021-06-03-ihs-base-image, Version: 9.0.20210930; Publisher: ibm-usa-ny-armonk-hq-6275750-ibmcloud-aiops, Offer: 2021-04-27-twas-cluster-base-image, Sku: 2021-04-27-twas-cluster-base-image, Version: 9.0.20210930. Please refer to https://docs.microsoft.com/en-us/azure/virtual-machines/windows/cli-ps-findimage for instructions on finding available images."}],"message":"The template deployment 'ibm-usa-ny-armonk-hq-6275750-ibmcloud-aiops.2021--20210930125304' is not valid according to the validation procedure. The tracking id is '1a0dbddf-32a8-4e29-b76f-91558ed25032'. See inner errors for details."}


@majguo @edburns Any suggestions on how to resolve this?

**Comment by git4rk** [comment]
We missed to click "Go Live" for the images.
After Graham did that, I was able to verify the preview.
Solution is ready for "Go Live"

**Comment by git4rk** [comment]
Graham submitted for "Go Live"

**Comment by git4rk** [comment]
Solution is live now.

**Comment by majguo** [comment]
I successfully verified deployments for both valid IBM ID and invalid IBM ID, as well as the image version:
```
dmgr: Uninstalled com.ibm.websphere.ND.v90_9.0.5009.20210823_0038 from the /datadrive/IBM/WebSphere/ND/V9/ directory. 
managed node: Uninstalled com.ibm.websphere.ND.v90_9.0.5009.20210823_0038 from the /datadrive/IBM/WebSphere/ND/V9/ directory.
ihs: Uninstalled com.ibm.websphere.IHS.v90_9.0.5009.20210823_0038 from the /datadrive/IBM/WebSphere/IHS/V9/ directory.
Uninstalled com.ibm.websphere.PLG.v90_9.0.5009.20210823_0038 from the /datadrive/IBM/WebSphere/Plugins/V9/ directory.
Uninstalled com.ibm.websphere.WCT.v90_9.0.5009.20210823_0038 from the /datadrive/IBM/WebSphere/Toolbox/V9/ directory.
```
All look good, closing the issue now.
